tests/utils.py

```diff
@@ -10,23 +10,14 @@ def unfix(n):
     return n // 10**18
 
 def longToHexString(value, leftPad=40):
-    hexStr = hex(value)[2:-1]
-    while len(hexStr) < leftPad:
-        hexStr = '0' + hexStr
-    if(hexStr[0:2] == '0x'):
-        return hexStr
-    else:
-        return '0x' + hexStr
+    # convert the value to a hex string, strip off the `0x`, strip off any trailing `L`, pad with zeros, prefix with `0x`
+    return '0x' + hex(value)[2:].rstrip('L').zfill(leftPad)
```

Comment:
Heh, I wouldn't be against breaking this out into multiple lines.  I was iterating on it a bunch trying to fix a number of bugs and by the time I was done this is what I had and breaking it out into multiple lines would require me to come up with variable names for each step... and it was late and my brain hurt.  :)

If people feel strongly (or at all) about breaking this up I can do that.  Open to help on the intermediate variable naming front.  ;)